### ISSUE TITLE
Can't launch mongodb instance with `--auth`

### ISSUE THREAD

**Issue by zhaoyi0113** [rueckstiess/mtools#646]
I use this command to launch a mongodb instance but failed on authentication setup.
```
mlaunch init --single --dir data/10652 --port 10652   --hostname localhost --auth
aunching: "mongod" on port 10652
Traceback (most recent call last):
  File "/Users/joeyzhao/.pyenv/versions/3.6.5/bin/mlaunch", line 11, in <module>
    sys.exit(main())
  File "/Users/joeyzhao/.pyenv/versions/3.6.5/lib/python3.6/site-packages/mtools/mlaunch/mlaunch.py", line 2042, in main
    tool.run()
  File "/Users/joeyzhao/.pyenv/versions/3.6.5/lib/python3.6/site-packages/mtools/mlaunch/mlaunch.py", line 536, in run
    getattr(self, self.args['command'])()
  File "/Users/joeyzhao/.pyenv/versions/3.6.5/lib/python3.6/site-packages/mtools/mlaunch/mlaunch.py", line 746, in init
    roles = self.args['auth_role']
KeyError: 'auth_role'
```

The mongodb instance is running but the authentication is not setup. The mongodb version I am using is 4.0.0. Python is 3.6.5. 

```
$ mlaunch --version
mtools version 1.5.0 || Python 3.6.5 (default, Jul 2 2018, 20:53:24) [GCC
4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
```

**Comment by stennie** [comment]
Thanks @zhaoyi0113, I can confirm the same error. Looking into a fix.

**Comment by stennie** [comment]
@zhaoyi0113 Looks like there was an off-by-one typo. Fixed this via 973c346.

Regards,
Stennie

**Comment by stennie** [comment]
@zhaoyi0113 Just released [mtools 1.5.1](https://github.com/rueckstiess/mtools/releases/tag/v1.5.1), so `pip install mtools --upgrade` should include this fix.

Regards,
Stennie
